src/txmempool.h

```diff
@@ -94,6 +89,25 @@ class CTxMemPoolEntry
     uint64_t nSizeWithAncestors;
     CAmount nModFeesWithAncestors;
     int64_t nSigOpCostWithAncestors;
+    size_t nTxWeight;          //!< ... and avoid recomputing tx weight (also used for GetTxSize())
+    size_t nModSize;           //!< ... and modified size for priority
+    size_t nUsageSize;         //!< ... and total memory usage
+
+public:
+    mutable size_t vTxHashesIdx; //!< Index in mempool's vTxHashes
+
+private:
+    unsigned int entryHeight;  //!< Chain height when entering the mempool
+
+    // Space that is typically used as padding. Commented out so that it doesn't cause extra memory use on different architectures.
+    // char PADDING_FREE_SPACE[2];
+
+    // Bool flags
+    struct CTxMemPoolEntryFlags {
+        //int UNUSED_BITS : 7;
+        bool spendsCoinbase : 1;       //!< keep track of transactions that spend a coinbase
+    };
+    CTxMemPoolEntryFlags flags;
```

Comment:
It's somewhat vestigial; iirc when I originally wrote this patch last year there was another bool that got removed in a later PR.

The only real remaining reason is to make the code "self documenting" so that when/if someone later adds another flag to mempool (or even adds the flag back), they are encouraged to dtrt rather than adding more memory bloat.